tests/appsec/iast_packages/packages/pkg_certifi.py

```diff
@@ -0,0 +1,28 @@
+"""
+certifi==2024.2.2
+
+https://pypi.org/project/certifi/
+"""
+from flask import Blueprint
+from flask import request
+
+from .utils import ResultResponse
+
+
+pkg_certifi = Blueprint("package_certifi", __name__)
+
+
+@pkg_certifi.route("/certifi")
+def pkg_certifi_view():
+    import certifi
+
+    response = ResultResponse(request.args.get("package_param"))
+
+    try:
+        # Ejemplo de uso común del paquete certifi
```

Comment:
forgive my ignorance, but should we have this in English?

Reply:
Absolutely (in my defense, this was on the previous PR that was removed and this built on, but should have reviewed everything).